### PR TITLE
Branch schema normalization python

### PR DESCRIPTION
**Pull request by sebalterego**
Schema normalization and fingerprint are missing for python.
I added 2 files for Schema normalization and associated unit tests. (Inspired from Java)

### CODE REVIEW

**Comment by busbey** [comment]
Implements AVRO-1318

File: lang/py/src/avro/schemaNormalization.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    def _build(self,jsonobj):
+        """
+        Create the canonical avro schema string (json) from a complete avro schema string
+        @arg json: json string representing an avro shema
+        @arg env: environment dict 
+        @return: canonical avro schema string
+        """
+        
+
+        #print jsonobj
```
Comment (busbey) [inline lang/py/src/avro/schemaNormalization.py]: Please either remove commented out lines or uncomment and expressly wrap in a debug flag od some kind.

**Comment by busbey** [comment]
Note to committer:

  - need to squash
  - need to fixup message
  - need to test generated fingerprints against other language.

**Comment by busbey** [comment]
Since this supports fingerprint64, and interop test would be a great addition that eases review.

File: lang/py/src/avro/schemaNormalization.py (hunk 1 above)
Comment (sebalterego) [inline lang/py/src/avro/schemaNormalization.py]: I think the 2 commented lines can be removed.

**Comment by sebalterego** [comment]
The test_schemaNormalization.py has the same unit tests as avro/lang/java/avro/src/test/java/org/apache/avro/TestSchemaNormalization.java


How do you implements interop tests with other language?

**Pull request closed without merging**